utils/build-script

```diff
@@ -340,13 +340,26 @@ details of the setups of other systems or automated environments.""")
         help="The host target. LLVM, Clang, and Swift will be built for this "
              "target. The built LLVM and Clang will be used to compile Swift "
              "for the cross-compilation targets.",
-        default=StdlibDeploymentTarget.host_target())
+        default=StdlibDeploymentTarget.host_target().name)
+    targets_group.add_argument(
+        "--cross-compile-hosts",
+        help="A space separated list of targets to cross-compile host Swift "
+             "tools for. Can be used multiple times.",
+        action="append", type=arguments.type.shell_split, default=[])
+    stdlib_targets = StdlibDeploymentTarget.default_stdlib_deployment_targets()
     targets_group.add_argument(
         "--stdlib-deployment-targets",
         help="list of targets to compile or cross-compile the Swift standard "
              "library for. %(default)s by default.",
         nargs="*",
-        default=StdlibDeploymentTarget.default_stdlib_deployment_targets())
+        default=[
+            target.name
+            for target in stdlib_targets])
+    targets_group.add_argument(
+        "--build-stdlib-deployment-targets",
```

Comment:
That's what we do at the moment. Any lipo-able hosts will get merged together unless you tell it to skip that step (again, should really be opt-in instead of opt-out, but only the Darwin hosts are lipo-able anyway)